go.mod

```diff
@@ -4,59 +4,53 @@ go 1.15
 
 require (
 	bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc
-	cloud.google.com/go/storage v1.8.0
-	contrib.go.opencensus.io/exporter/prometheus v0.1.0
-	github.com/Azure/azure-pipeline-go v0.2.2 // indirect
-	github.com/Azure/azure-storage-blob-go v0.8.0
+	cloud.google.com/go/storage v1.12.0
+	contrib.go.opencensus.io/exporter/prometheus v0.2.0
+	github.com/Azure/azure-pipeline-go v0.2.3 // indirect
+	github.com/Azure/azure-storage-blob-go v0.10.0
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
-	github.com/alexhunt7/ssher v0.0.0-20190216204854-d36569cf7047
-	github.com/aws/aws-sdk-go v1.31.3
+	github.com/aws/aws-sdk-go v1.34.29
 	github.com/bgentry/speakeasy v0.1.0
 	github.com/chmduquesne/rollinghash v4.0.0+incompatible
 	github.com/efarrer/iothrottler v0.0.1
 	github.com/fatih/color v1.9.0
 	github.com/foomo/htpasswd v0.0.0-20200116085101-e3a90e78da9c
-	github.com/gofrs/flock v0.7.1
+	github.com/gofrs/flock v0.8.0
 	github.com/golang/protobuf v1.4.2
 	github.com/google/fswalker v0.2.1-0.20200214223026-f0e929ba4126
-	github.com/google/go-cmp v0.4.0
+	github.com/google/go-cmp v0.5.2
 	github.com/google/readahead v0.0.0-20161222183148-eaceba169032 // indirect
-	github.com/google/uuid v1.1.1
-	github.com/google/wire v0.4.0 // indirect
-	github.com/gorilla/mux v1.7.4
-	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd // indirect
-	github.com/klauspost/compress v1.10.10
-	github.com/klauspost/pgzip v1.2.4
+	github.com/google/uuid v1.1.2
+	github.com/gorilla/mux v1.8.0
+	github.com/klauspost/compress v1.11.0
+	github.com/klauspost/pgzip v1.2.5
 	github.com/kylelemons/godebug v1.1.0
-	github.com/mattn/go-colorable v0.1.6 // indirect
-	github.com/mattn/go-ieproxy v0.0.1 // indirect
-	github.com/minio/minio v0.0.0-20200521180750-fade0562443f
-	github.com/minio/minio-go/v6 v6.0.55
-	github.com/natefinch/atomic v0.0.0-20150920032501-a62ce929ffcc
+	github.com/mattn/go-colorable v0.1.7 // indirect
+	github.com/minio/minio v0.0.0-20200922223427-0537a21b7948
+	github.com/minio/minio-go/v6 v6.0.58-0.20200612001654-a57fec8037ec
```

Comment:
can we upgrade to v7?

also, can we get rid of minio/minio which is only used in tests.

Reply:
`go mod` will include dependencies even if they are only used in test or in a different platform.
I can push a separate patch to upgrade to v7, in case there are any issues with it to at least upgrade the rest.

Reply:
The test depends on this "github.com/minio/minio/pkg/madmin" - I was wondering whether there's equivalent in v7 package.